docs/install_guides/configurations-for-intel-gpu.md

```diff
@@ -7,37 +7,21 @@
 
 ## Linux
 
-If you have installed OpenVINO Runtime from the archive file, APT, or YUM, follow these steps to work with GPU:
+To install the latest available **Intel® Graphics Compute Runtime for OpenCL™** for your OS, see the [Install Guides](https://github.com/intel/compute-runtime/releases/latest).
+> **NOTE**: For instructions specific for discrete graphics platforms (Intel® Arc™ A-Series Graphics, Intel® Data Center GPU Flex Series and Intel® Data Center GPU MAX Series, Intel® processor graphics Gen12, Intel® Iris Xe MAX (codename DG1)), please refer to [this guide](https://dgpu-docs.intel.com/installation-guides/index.html).
 
-1. Go to the install_dependencies directory:
-   ```sh
-   cd <INSTALL_DIR>/install_dependencies/
-   ```
+Alternatively, you can install one of the earlier versions of the driver based on your internal considerations.
 
-2. Install the **Intel® Graphics Compute Runtime for OpenCL™** driver components required to use the GPU plugin and write custom layers for Intel® Integrated Graphics. The drivers are not included in the package. To install it, run this script:
-   ```sh
-   sudo -E ./install_NEO_OCL_driver.sh
-   ```
-   > **NOTE**: To use the **Intel® Iris® Xe MAX Graphics**, see the [Intel® Iris® Xe MAX Graphics with Linux*](https://dgpu-docs.intel.com/devices/iris-xe-max-graphics/index.html) page for driver installation instructions.
-   
-   The script compares the driver version on the system to the current version. If the driver version on the system is higher or equal to the current version, the script does 
-   not install a new driver. If the version of the driver is lower than the current version, the script uninstalls the lower version and installs the current version with your permission:
-   ![](../img/NEO_check_agreement.png) 
-
-   Higher hardware versions require a higher driver version, namely 20.35 instead of 19.41. If the script fails to uninstall the driver, uninstall it manually. During the script execution, you may see the following command line output:  
-   ```sh
-   Add OpenCL user to video group    
-   ```
-   Ignore this suggestion and continue.<br>
-   You can also find the most recent version of the driver, installation procedure and other information on the [Intel® software for general purpose GPU capabilities](https://dgpu-docs.intel.com/index.html) site.
-
-3. **Optional:** Install header files to allow compilation of new code. You can find the header files at [Khronos OpenCL™ API Headers](https://github.com/KhronosGroup/OpenCL-Headers.git).
+Additionally, it is recommended that you refer to the [Intel® Graphics Compute Runtime Github page](https://github.com/intel/compute-runtime/) for the instructions and recommendations on GPU driver installation specific to particular release, including the list of supported hardware platforms.
 
 You've completed all required configuration steps to perform inference on processor graphics.
+
```

Comment:
question: if I follow all these instruction, will they recommend me to install only driver or OpenCL library as well?
Old script did both steps

Reply:
I've checked it on ubu20 and rhel8
### Ubuntu20
sudo -E apt-cache show intel-opencl-icd
Package: intel-opencl-icd
Provides: opencl-icd
Depends: libc6 (>= 2.29), libgcc-s1 (>= 3.4), libigdgmm11 (>= 20.1.1+ds1), libstdc++6 (>= 9), libigc1 (>= 1.0.3627), libigdfcl1 (>= 1.0.3627), ocl-icd-libopencl1
### Rhel8
Need to be added additional step with installing http://mirror.centos.org/centos/8-stream/AppStream/x86_64/os/Packages/ocl-icd-2.2.12-1.el8.x86_64.rpm
I'll also notify Driver team about such difference